sql/telemetry_derived/clients_daily_active_hours_by_country_aggregates_v1/query.sql

```diff
@@ -0,0 +1,42 @@
+WITH geo_t AS (
+  SELECT
+    client_id,
+    geo
+  FROM
+    (
+      SELECT
+        client_id,
+        country AS geo,
+        ROW_NUMBER() OVER (PARTITION BY client_id ORDER BY submission_date DESC) AS rn
```

Comment:
I don't entirely understand the machinery here, but I think this will cause a problem.  The geo for a profile in the pipeline needs to be fixed over time, as updated to the IPGeo database cause users to "teleport" between locations sometimes, introducing major anomalies at times.

Given that we're considering moving to a "clients_first_seen" notion of profile creation, it might be worth basing this on a such a table that would capture the basic dimensions (including geo) of each profiles first (ever) ping.

Reply:
I'm tempted to use this as motivation to go ahead and create a clients_first_seen table. We can probably make that happen this week if it seems like the right thing to do for this pipeline.;

Reply:
What about `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v6`?

Reply:
I just created that today 😆 

I was testing out a possible design for the query to serve this need. I'll try to turn that into a PR right now.

Reply:
See https://github.com/mozilla/bigquery-etl/pull/952